app/models/AddressLine.scala

```diff
@@ -29,9 +29,10 @@ sealed trait AddressLine {
 object AddressLine {
 
   case object AddressLine1 extends AddressLine {
-    override val field: String = "addressLine1"
-    val length: Int            = 35
-    val regex: Regex           = stringFieldRegex
+    override val field: String  = "addressLine1"
+    val length: Int             = 35
+    val streetNumberLength: Int = 6
```

Comment:
I would be tempted to create a different case object for `StreetNumber`. From the messages it also looks like the regex should only allow numbers?

Reply:
Gavin and I had a discussion regarding, some streets numbers may have letters in them so I've left it as it is for now. Will put it in a separate object

Reply:
@BledarFeratiEY that's a fair point. We may need to amend the message key in that case as it's currently saying "The street number must only include numbers 0 to 9". Happy to approve before that decision gets made though.